packages/big-design/src/components/Collapse/styled.tsx

```diff
@@ -0,0 +1,35 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { withTransition } from '../../mixins/transitions';
+import { StyleableButton } from '../Button/private';
+import { H3 } from '../Typography';
+
+export const StyledTitle = styled(H3)`
+  button {
+    width: auto;
+  }
+`;
+
+export const StyledButton = styled(StyleableButton)`
+  padding-left: ${({ theme }) => theme.spacing.none};
+  padding-right: ${({ theme }) => theme.spacing.none};
+
+  &:hover:not(:active) {
+    background: none;
+  }
```

Comment:
Can you confirm with design that there are no other requirements around this? A couple cases that need to be considered:
- mobile (default functionality for buttons expands it to full-width)
- spacing
  - around the component (both inside/outside the box-model)
  - spacing between the icon and text
- hover/focus/active styles 

Just trying to make sure that this component is 💅before it goes to design for review.

Reply:
I discussed and confirm these moments. And made changes.